BDKSwiftExampleWallet/View/SettingsView.swift

```diff
@@ -34,29 +30,21 @@ struct SettingsView: View {
                                 with: ""
                             )
                         )
+                    } else {
+                        HStack {
+                            Text("Disconnected")
```

Comment:
What about "Not configured" or "Unavailable"?

Reply:
if the `getNetwork` function doesn't return anything (which would be a rare thing) we should probably just put "No Network".

But you can see I just previously didn't show anything because that would be such a rare circumstance where it didn't which of the four possible networks you were on: https://github.com/bitcoindevkit/BDKSwiftExampleWallet/blob/b946ba4caaac8ef2493c8cc3869ca5acdec0efa7/BDKSwiftExampleWallet/View/SettingsView.swift#L26C3-L37C22

But if you do want to show something, which I'm ok with, via adding that `else` clause` it would be "No Network".